.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,15 @@
+name: Shellcheck lint
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run Shellcheck
+        uses: sudo-bot/action-shellcheck@latest

```

Comment:
Let's pin the version here to avoid breakage, Dependabot will bump it for us

Reply:
This action only has this version...

Reply:
That's unfortunate, I think I would rather use https://github.com/ludeeus/action-shellcheck then, it seems like it has more users and it does follow semantic versioning when tagging. Would you mind changing the action used to that one?